starknet_py/utils/crypto/facade.py

```diff
@@ -46,3 +48,7 @@ def pedersen_hash(left: int, right: int) -> int:
     if use_cpp_variant():
         return cpp_hash(left, right)
     return default_hash(left, right)
+
+
+def get_random_salt() -> int:
+    return random.Random().randrange(0, FIELD_PRIME)
```

Comment:
This doesn't have anything with crypto, it doesn't use entropy based random values now. I think we can move it somewhere else.

Reply:
Does everyone agree that `get_random_salt` should be moved to the `udc_deployer/deployer.py` file? I'm quite unsure, but this is the only place where we use it (outside the tests)